db/memtable_list.cc

```diff
@@ -729,6 +744,19 @@ Status InstallMemtableAtomicFlushResults(
   // this can release and reacquire the mutex.
   s = vset->LogAndApply(cfds, mutable_cf_options_list, edit_lists, mu,
```

Comment:
@riversand963  `InstallMemtableAtomicFlushResults` does not consider setting `min_log_number_to_keep` in 2pc mode, is it because it does not support 2pc?